templates/generic.py

```diff
@@ -26,9 +27,10 @@ class GenericTemplate:
     def __init__(self):
         self.template = copy(template['value'])
         self.elements = []
-    def add_element(self, title='', item_url='', image_url='', subtitle='', buttons=[]):
+    def add_element(self, title='', description='', item_url='', image_url='', subtitle='', buttons=[]):
         element = {}
         element['title'] = title[:TITLE_CHARACTER_LIMIT]
+        element['description'] = description[:DESCRIPTION_CHARACTER_LIMIT]
```

Comment:
My Understanding was that GenericTemplate is something we have written as a wrapper for formatting the output we are generating for each module. Can you please tell me about it?

Also, I din't understand the meaning of adding attribution.

Would be great if you could please help with the above doubts :)